components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/OpcBaseNodeModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Publisher.Config.Models {
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Describing a base entry in a node list
+    /// </summary>
+    [DataContract]
+    public abstract class OpcBaseNodeModel {
```

Comment:
>public abstract class OpcBaseNodeModel [](http://example.com/codeflow?start=4&length=38)

Adding this class is a bit confusing. We now have a OpcBaseNodeModel class and an OpcNodeModel class that does not inherit from the base class model. There is also an OpcDataNodeModel class that inherit from the base model, but it's never used.